Source/OxyPlot/Series/BarSeries/LinearBarSeries.cs

```diff
@@ -32,15 +32,15 @@ public class LinearBarSeries : DataPointSeries
         private OxyColor defaultColor;
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="LinearBarSeries" /> class.
+        /// Initializes a new instance of the <see cref="LinearBarSeries"/> class.
         /// </summary>
         public LinearBarSeries()
         {
             this.FillColor = OxyColors.Automatic;
             this.BarWidth = 5;
             this.StrokeColor = OxyColors.Black;
             this.StrokeThickness = 0;
-            this.TrackerFormatString = XYAxisSeries.DefaultTrackerFormatString;
+            this.TrackerFormatString = DefaultTrackerFormatString;
```

Comment:
There are OK, but again, if there's a way to turn them off, I think it's clearer as it was.